containers-tests/tests/intmap-properties.hs

```diff
@@ -307,6 +308,20 @@ test_lookup = do
         country <- lookup dept deptCountry
         lookup country countryCurrency
 
+test_query :: Assertion
+test_query = do
+    employeeCurrency 1 @?= Just 1
+    employeeCurrency 2 @?= Nothing
+  where
+    employeeDept = fromList([(1,2), (3,1)])
+    deptCountry = fromList([(1,1), (2,2)])
+    countryCurrency = fromList([(1, 2), (2, 1)])
```

Comment:
```suggestion
    employeeDept = fromList [(1, 2), (3, 1)]
    deptCountry = fromList [(1, 1), (2, 2)]
    countryCurrency = fromList [(1, 2), (2, 1)]
```
Seems you copied this as well, but the parentheses are just unnecessary (Haskell functions are not called like `f(...)` after all).

Reply:
Yes, I just kept what was in place, I'll change both and extend the test as well a small amount.

Reply:
Let me know if you would like any changes to the new test inputs.